### DISCUSSION TITLE
should i sync these different firefox runtime to my firefox sync ?

### DISCUSSION THREAD

**Discussion by as3mbus** [filips123/PWAsForFirefox#294] [Q&A] [answered]
first of all, I'm grateful for this plugin as it addresses my need for a single web app window through firefox.

but I have several questions that are nagging me after installing everything. 

can I sync this different firefox browser, with firefox sync? to synchronize my plugin?
why does this application need to have different firefox runtime to begin with?

I'm sorry for being suspicious of this plugin. since a browser is a crucial security point that can be dangerous if compromised

**Comment by filips123** [comment, marked as answer]
> can I sync this different firefox browser, with firefox sync? to synchronize my plugin?

You should be able to sync everything that is supported by Firefox Sync. However, maybe disable settings sync, because you might need to have different settings for the app browser than for the normal one.

> why does this application need to have different firefox runtime to begin with?

Because it needs to apply [certain code](https://github.com/filips123/PWAsForFirefox/tree/main/native/userchrome) to modify Firefox to behave like a single web app. Theoretically, it could probably also work by modifying the main Firefox, but I choose not to do this, as it may cause instability in case something breaks. Additionally, using separate runtime and profiles also allows you to have different settings in web apps than in the main Firefox.
